source/reference/authentication.txt

```diff
@@ -0,0 +1,116 @@
+.. _c2c-authentication:
+
+=================================================
+Authentication Using Workload Identity Federation
+=================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+The source and destination clusters must use MongoDB 7.0 or later.
+
+You can use ``mongosync`` to sync data between source and destination
+clusters and authenticate the cluster connections using various
+combinations:
+
+- Sync source and destination clusters both authenticated using Workload
+  Identity Federation.
+- Sync a source cluster authenticated using Workload Identity Federation
+  and a destination cluster that doesn't use Workload Identity
+  Federation.
+- Sync a destination cluster authenticated using Workload Identity
+  Federation and a source cluster that doesn't use Workload Identity
+  Federation.
+
+``mongosync`` supports replica sets and sharded clusters.
```

Comment:
Should specifically mention OIDC-authenticated connections? (The fact that “mongosync supports replica sets and sharded clusters” per se is not specifically germane to OIDC, after all.)